bedrock/firefox/templates/firefox/concerts.html

```diff
@@ -50,40 +36,33 @@
-            </a>
-          {% elif switch('firefox_concert_one_over') %}
-            <img src="{{ static('img/firefox/concerts/video-poster-phosphorescent.jpg') }}" alt="Phosphorescent (Matthew Houck) and his six-piece band perform on stage before a crowd. The lighting is tinted purple and the band is backed by a large, illuminated sign reading ‘C’est La Vie’, the title of their recent album.">
-          {% else %}
-            <a class="video-play js-video-play" href="https://youtu.be/PcNpDCdkhLc" data-id="PcNpDCdkhLc" title="Play the video">
-              <img src="{{ static('img/firefox/concerts/video-poster-phosphorescent.jpg') }}" alt="Phosphorescent (Matthew Houck) and his six-piece band perform on stage before a crowd. The lighting is tinted purple and the band is backed by a large, illuminated sign reading ‘C’est La Vie’, the title of their recent album.">
+            <a class="video-play js-video-play" href="https://youtu.be/xD2rHkqLHuw" data-id="xD2rHkqLHuw" title="Play the video">
+              <img src="{{ static('img/firefox/concerts/concert-phosphorescent.jpg') }}" alt="Phosphorescent (Matthew Houck) and his six-piece band perform on stage before a crowd at the 9:30 Club on December 11, 2018. The lights are tinted purple and gold and the band is backed by a large, illuminated sign reading ‘C’est La Vie’, the title of their recent album.">
             </a>
-          {% endif %}
           </div>
         </div>
         <div class="mzp-c-card-feature-content">
           <div class="mzp-c-card-feature-content-container">
-          {% if switch('firefox_concert_one_live') %}
-            <p class="mzp-c-card-tag live">Live</p>
-          {% elif switch('firefox_concert_one_over') %}
-            <p class="mzp-c-card-tag ended">Coming soon</p>
-          {% else %}
-            <p class="mzp-c-card-tag countdown" id="countdown-one">Starts in <span class="time days"></span> days, <span class="time hours"></span> hours</p>
-          {% endif %}
-
-          {% if switch('firefox_concert_one_over') %}
+            <p class="mzp-c-card-tag ended">Dec. 11, 2018</p>
+            <h2 class="mzp-c-card-feature-title">Phosphorescent</h2>
+            <p class="mzp-c-card-feature-desc">On December 11 the Firefox Concert Series livestreamed one of the last shows on Phosphorescent’s <cite lang="fr">C’est La Vie</cite> tour from DC’s 9:30 Club. Enjoy this exclusive clip.</p>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <div class="artist-next mzp-l-content l-content-narrow">
+      <section class="artist-card mzp-c-card-feature mzp-has-aspect-16-9 mzp-l-card-feature-right-half mzp-t-firefox mzp-t-dark">
```

Comment:
Not a blocker as it wasn't introduced in this PR, but we should try and avoid creating components using the Protocol naming conventions that don't exist imho. These should likely drop the `mzp-` namespace.

Reply:
Oh, brain malfunction! I'm totally mistaken here and this does exist. For some reason I thought the feature card had a different naming convention. I was confused as this wasn't using the feature card macro, so I assumed it was something custom. My bad!

Reply:
Yeah, sorry, I didn't use the macro because... some reason... there was some piece of it that did need to be custom but now I forget what it was. The countdown maybe. But it's a slightly modified feature card.